test/e2e/apps/statefulset.go

```diff
@@ -188,7 +188,7 @@ var _ = SIGDescribe("StatefulSet", func() {
 			)).To(Succeed(), "wait for pod %q to be released", pod.Name)
 
 			// If we don't do this, the test leaks the Pod and PVC.
-			By("Readding labels to the stateful set's pod")
```

Comment:
As per the code below, this seems like not a typo. It meant to say `re-add`, IMO.